webtau-core/src/main/java/com/twosigma/webtau/WebTauCore.java

```diff
@@ -43,11 +43,9 @@
 import java.util.regex.Pattern;
 
 /**
- * Data Driven Java Testing
- * <p>
- * Convenient class for a single static * imports
+ * Convenient class for a single static * imports to have matchers and helper functions available for your test
  */
-public class Ddjt {
+public class WebTauCore extends Matchers {
```

Comment:
Are we ok with this back compat break?

Reply:
I've re-read the description of the PR.  I'm OK with this break I think because I doubt anyone other than Clemens is using it.

Reply:
I can't guarantee that, I've been showing DDJT to others -- just mention it in the release notes / readme as well as suggested (non-)usage.